src/umessage/umessagebuilder.rs

```diff
@@ -362,7 +368,14 @@ impl UMessageBuilder {
         {
             assert!(priority.value() >= UPriority::UPRIORITY_CS4.value())
         }
-        self.priority = priority;
+        if priority != PRIORITY_DEFAULT {
```

Comment:
Does that mean if the priority is default, then we don't need to include the priority in the serialized protobuf?

Reply:
What if priority equals to `UPriority::UPRIORITY_UNSPECIFIED`?

Reply:
> Does that mean if the priority is default, then we don't need to include the priority in the serialized protobuf?

According to https://github.com/eclipse-uprotocol/up-spec/blob/main/basics/qos.adoc I would say so, yes.

Reply:
I kind of got your point.

Reply:
> What if priority equals to `UPriority::UPRIORITY_UNSPECIFIED`?

Then the priority will not be included in the serialized protobuf because it has the default value (0 in this case).